planning/behavior_path_planner/include/behavior_path_planner/behavior_path_planner_node.hpp

```diff
@@ -68,6 +69,7 @@ using geometry_msgs::msg::TwistStamped;
 using nav_msgs::msg::OccupancyGrid;
 using nav_msgs::msg::Odometry;
 using rcl_interfaces::msg::SetParametersResult;
+using std_msgs::msg::Bool;
```

Comment:
@purewater0901  `std_msgs` is deprecated. Please consider using a specific message for route availability.

Reply:
doen with https://github.com/autowarefoundation/autoware.universe/pull/4543/commits/c80932b56ce833a048aa1961ae493f3ecba255ae